tests/test_commands.py

```diff
@@ -125,3 +135,60 @@ def test_validation_failed_message(self):
         self.assertIn("user", output.getvalue())
         self.assertIn("783", output.getvalue())
         self.assertIn("does not exist", output.getvalue())
+
+
+class ClearTokensTest(TestCase):
+    def setUp(self):
+        now = timezone.now()
+        earlier = now - timedelta(seconds=100)
+        later = now + timedelta(seconds=100)
+        UserModel = get_user_model()
+        user = UserModel.objects.create()
+
+        app = Application.objects.create(
+            client_type="confidential",
+            authorization_grant_type="client-credentials",
+            name="The App",
+        )
+        for i in range(100):
+            old = AccesstokenModel.objects.create(token="old access token {}".format(i), expires=earlier)
+            new = AccesstokenModel.objects.create(token="current access token {}".format(i), expires=later)
+            # make half of these Access Tokens have related Refresh Tokens, which prevent their deletion.
+            if i % 2:
+                RefreshtokenModel.objects.create(
```

Comment:
Use bulk_create. Use list comprehensions with RefreshtokenModel(...) and run RefreshtokenModel.objects.bulk_create The order in id increment should remain the same.